libclc/clc/include/clc/integer/clc_bitfield_extract.inc

```diff
@@ -0,0 +1,10 @@
+//===----------------------------------------------------------------------===//
```

Comment:
It's minor but I'd perhaps suggest naming this file `clc_bitfield_extract_decl.inc` as it's a helper file for differently named declarations, and not declaration a builtin called `__clc_bitfield_extract`.

Reply:
done, thanks. Renamed clc_bitfield_extract.inc to clc_bitfield_extract_decl.inc, bitfield_extract.inc to bitfield_extract_decl.inc / bitfield_extract_def.inc

Reply:
Thanks!